File: MiniDOMFinanziaria/Finanziaria/Preventivi/Offerte/Evaluators/CCQSPDProvvigioneXOffertaEvaluator.cs

namespace minidom
{
    public partial class Finanziaria
    {
        public abstract class CCQSPDProvvigioneXOffertaEvaluator
        {
            /// <summary>
            /// Valuta il valore delle estinzioni
            /// </summary>
            /// <param name="pxo"></param>
            /// <param name="offerta"></param>
            /// <param name="estinzioni"></param>
            /// <returns></returns>
            public abstract decimal Evaluate(CCQSPDProvvigioneXOfferta pxo, COffertaCQS offerta, CCollection<EstinzioneXEstintore> estinzioni);
        }
    }
}

File: MiniDOMFinanziaria/Finanziaria/Preventivi/Offerte/Evaluators/CCQSPDProvvigioneXOffertaEvaluator.vb
Imports minidom
Imports minidom.Databases
Imports minidom.Sistema
Imports minidom.Anagrafica



Partial Public Class Finanziaria


    Public MustInherit Class CCQSPDProvvigioneXOffertaEvaluator

        Public MustOverride Function Evaluate(ByVal pxo As CCQSPDProvvigioneXOfferta, ByVal offerta As COffertaCQS, ByVal estinzioni As CCollection(Of EstinzioneXEstintore)) As Double

    End Class

End Class
